### ISSUE TITLE
[Bug] Syntax errors with workflow conditional expressions, needing to reset step to 0

### ISSUE THREAD

**Issue by andyfeller** [skills/continuous-delivery-azure#8]
**Describe the bug**
1. **Workflow step state is non-`0`**

   It appears that previous development on the template did not revert https://github.com/skills/continuous-delivery-azure/blob/main/.github/script/STEP back to `0`

1. **Invalid YAML for multi-line conditional**

   Using all of the examples above, this is invalid syntax for [block YAML use cases](https://yaml.org/spec/1.2.2/#chapter-8-block-style-productions).

   Instead of:

   ```yaml
         if: ${{ github.repository_owner != 'TBD-organization'
           && needs.get_current_step.outputs.current_step == 6 }}
   ```

   One of the following forms must be used:

   1. Single line

      ```yaml
            if: ${{ github.repository_owner != 'TBD-organization' && needs.get_current_step.outputs.current_step == 6 }}
      ```

   1. Multi line block

      ```yaml
            if: |
              ${{ github.repository_owner != 'TBD-organization'
                  && needs.get_current_step.outputs.current_step == 6 }}
      ```

**To Reproduce**

**Expected behavior**

**Screenshots**

**Device information**
 - Type: [e.g. iPhone6]

**Additional context**

**Comment by andyfeller** [comment]
cc: @emilyistoofunky

**Comment by markpatterson27** [comment]
PR #6 fixes this

**Comment by sinsukehlab** [comment]
I will fix YAML style in PR #6.

**Comment by andyfeller** [comment]
Closing this out as #6 should have fixed this
